pkg/registry/apis/provisioning/list.go

```diff
@@ -0,0 +1,72 @@
+package provisioning
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/registry/rest"
+
+	provisioning "github.com/grafana/grafana/pkg/apis/provisioning/v0alpha1"
+	"github.com/grafana/grafana/pkg/registry/apis/provisioning/resources"
+)
+
+type listConnector struct {
+	getter RepoGetter
+	lister resources.ObjectLister
+}
+
+func (*listConnector) New() runtime.Object {
+	return &provisioning.TestResults{}
```

Comment:
Shouldn't this be a `*provisioning.ObjectList`?

Reply:
🤦🏻